docs/src/app/components/pages/components/Tabs/Page.jsx

```diff
@@ -10,24 +10,43 @@ import TabsExampleControlled from './ExampleControlled';
 import tabsExampleSwipeableCode from '!raw!./ExampleSwipeable';
 import TabsExampleSwipeable from './ExampleSwipeable';
 import tabsCode from '!raw!material-ui/lib/tabs/tabs';
-import tabsText from './Tabs';
 import tabCode from '!raw!material-ui/lib/tabs/tab';
-import tabText from './Tab';
+
+const descriptions = {
+  simple: 'A simple example of Tabs. The third tab demonstrates the `onActive` property of `Tab`.',
+  controlled: 'An example of controlled tabs. The selected tab is handled through state and callbacks in the parent ' +
+  '(example) component.',
+  swipeable: 'This example integrates the [react-swipeable-views]' +
+  '(https://github.com/oliviertassinari/react-swipeable-views) component with Tabs, animating the Tab transition, ' +
+  'and allowing tabs to be swiped on touch devices.',
```

Comment:
How about using template string for these long descriptions? Makes it easier to edit :grin: